### DISCUSSION TITLE
Vroid SDK Authorization Code being requested too often

### DISCUSSION THREAD

**Discussion by Shinnonda** [pixiv/vroid-sdk-developers#53] [Q&A]
I noticed while developing that the AuthCode is requested too often while testing, (4/5 times per hour testing).
Obviously i'm starting and stopping the game all the time on editor.
Thing is, i'm thinking how this will behave since the game is multiplayer, and the player will receive new licenses all the time, imagine a new player joining the same server than you, and it just pop your browser up in the middle of your gameplay to make you log-in on pixiv again.
(i leaved the game running for a while to see if it gonna request again when received a new license,and it did).

edit:
I just noticed that Craftopia doesn't ask you for login all the time, so i just don't know how to do it properly then.
PS: it would be interessting to have the call for the auth code ingame as well, just to make simpler for the user.

**Comment by nyamadan** [comment]
I think the account file may be broken.
Once you enter the AuthCode, it will not be requested again.
Could you delete the files in [Application.persistentDataPath](https://docs.unity3d.com/2019.4/Documentation/ScriptReference/Application-persistentDataPath.html)?

> PS: it would be interessting to have the call for the auth code ingame as well, just to make simpler for the user.

Thanks for your suggestion.

Have you tried the [Loopback Redirect Interface](https://vroid.notion.site/From-0-0-21-to-0-1-0-3d79655e76764441a598dfa8f6f770b8#a363c9349ce74644a7a4ac214a22911d)?
It will allow you to log in without entering the AuthCode.

**Reply by Shinnonda** [reply]
oh! i didn't tried that yet. i'll look it up, thank you!

edit:
Loopback already make it way less complicated. But still popping up the Vroid auth page frequently.
I've cleaned Persistent Data, still asking really often, i also tried out in another computer, same thing happend.
